spec/requests/api/resources_controller_spec.rb

```diff
@@ -207,4 +214,94 @@
       end
     end
   end
+
+  path '/api/resources/{resource_id}/completed' do
+    get 'Return Resource completed status' do
+      tags 'Resources'
+      description 'Retrieves status of the resource for the actual user'
+      parameter name: :resource_id, in: :path, type: :string
+      produces 'application/json'
+      operationId 'getResourceStatus'
+
+      let(:resource_id) { create(:resource, name: 'ruby_course').id }
```

Comment:
Detail: ¿Why specify "name"? The factory could manage without input.

Reply:
It is true. I copied the spec of learning units.